### DISCUSSION TITLE
How do you flip the screen?

### DISCUSSION THREAD

**Discussion by TricksterSam** [djphazer/O_C-Phazerville#41] [General]
I uncommented line 20 in OC_options.h, but the screen is still inverted.

```
/* ------------ flip screen / IO mapping ------------------------------------------------------------  */
#define FLIP_180
```

Is there a different way to make this happen in phazerville?

**Comment by djphazer** [comment]
I've been turning flags like FLIP_180 on in platformio.ini instead. Not sure why it wouldn't work by uncommenting in OC_options though...

**Reply by TricksterSam** [reply]
Thanks. That was exactly what I needed.
